extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -534,21 +534,25 @@ export class ProjectsController {
 
 		const result: mssql.SchemaComparePublishProjectResult = await service.schemaComparePublishProjectChanges(operationId, projectPath, fs, utils.getAzdataApi()!.TaskExecutionMode.execute);
 
-		const project = await Project.openProject(projectFilePath);
+		result.success = result.success && result.errorMessage === '';
```

Comment:
it seems a little strange to be reseting the value of result.success instead of having a new variable or having the if condition be checking result success and no error message

Reply:
Given the spot of code you found, it seems that `result.success` being true is by design, I'll key off of the error message.  I redefined the success bool because I was hedging my bets that that was wrong.